big-o-notation/big-o.js

```diff
@@ -0,0 +1,44 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {};                  // 1 * n = O(n)
+  const unique = [];                // 1 * n = O(n)
+  for (
+    let i = 0;                      // 1 * n = O(n)
+    i < words.length;               // 2 * n = O(2n) ~= 0(n)
+    i++                             // 2 * n = O(2n) ~= 0(n)
+  ) {
+    const word = words[i];          // 3 * n = O(3n) ~= 0(n)
+    if (!seen[word]) {              // 2 * n = O(2n) ~= 0(n)
+      seen[word] = true;
+      unique[unique.length] = word; // 3 * n = O(3n) ~= 0(n)
```

Comment:
This is actually only 2 operations, accessing the `length` property and the assignment